btree/btree.c

```diff
@@ -1171,9 +1182,12 @@ struct level {
 	/** nd for newly allocated node at the level. **/
 	struct nd *l_alloc;
 
-	/** flag for indicating if node needs to get freed. **/
+	/**
+	 * This is the flag for indicating if node needs to be freed. Currently
+	 * this flag is set in delete operation and is used by P_FREENODE phase
+	 * for determine if node needs to be freed.
```

Comment:
s/for determine if node needs to be freed/to determine if the node should be freed/

Reply:
done